utilities/stage_release.sh

```diff
@@ -1,21 +0,0 @@
-#!/bin/bash
```

Comment:
What will we need to run instead of `stage_release.sh`?

Reply:
`stage_sites.py` and then `mvn deploy` directly or a one-line script (I haven't decided yet). Those deployment scripts (`deploy.sh`, `stage_release.sh`, `create_site.sh`, `finalize_release.sh`) are doing mostly nothing or things which are obsolete now (like figuring global version for site or deploying snapshot  to sonatype (which we have never done and I belive will never do, because this is not how we "release" snapshots for people who may be interested in them)). 

I'm also thinking about just putting everythin in one script. In general I wan tot make the whole release a one-click thing, but need to handle the version bump thing first, that is why not doing it now. I'll specify all that in RELEASING.md once release is done.